omnisharp/test.json

```diff
@@ -5,6 +5,9 @@
   "versionSpecific": false,
   "type": "bash",
   "cleanup": true,
+  "skipWhen": [
+    "version=8" // see https://github.com/redhat-developer/dotnet-regular-tests/issues/250
```

Comment:
Could you remove https://github.com/redhat-developer/dotnet-regular-tests/blob/43c34b5608b0957301a03b79b77b832084db8328/omnisharp/test.sh#L22-L25 and add it here too?

Reply:
#done